src/core/ConversationModel.cpp

```diff
@@ -277,7 +277,11 @@ QVariant ConversationModel::data(const QModelIndex &index, int role) const
     const MessageData &message = messages[index.row()];
 
     switch (role) {
-        case Qt::DisplayRole: return message.text;
+        case Qt::DisplayRole:
+          if (message.text.startsWith(QStringLiteral("/me "))) {
```

Comment:
Use QLatin1String here

(wherever Qt APIs have a QLatin1String overload, prefer using it to QString or QStringLiteral, as the former doesn't do any unicode conversion, and is thus less expensive)